src/plugins/walletConnectToDapps/components/ExplorationBanner.tsx

```diff
@@ -0,0 +1,44 @@
+import { Avatar, Box, Button, Flex, Image, Stack, useColorModeValue } from '@chakra-ui/react'
+import bannerImg from 'assets/dapps-banner.png'
+import { WalletConnectCurrentColorIcon } from 'components/Icons/WalletConnectIcon'
+import { Text } from 'components/Text'
+import { FC } from 'react'
+
+export const ExplorationBanner: FC = () => (
+  <Box
+    borderWidth={1}
+    borderColor={useColorModeValue('blackAlpha.50', 'gray.750')}
+    borderRadius='lg'
+  >
+    <Stack direction='row' spacing={4}>
+      <Flex flex={1}>
+        <Image objectFit='cover' boxSize='100%' src={bannerImg} />
+      </Flex>
+      <Stack flex={2} alignSelf='center' spacing={4} p={8}>
+        <Avatar
+          bg='gray.700'
+          icon={<WalletConnectCurrentColorIcon color={useColorModeValue('blue.500', 'blue.400')} />}
+        />
+
+        <Box>
+          <Text as='b' fontSize='lg' translation='plugins.walletConnectToDapps.banner.title' />
+          <Text translation='plugins.walletConnectToDapps.banner.subtitle' color='gray.500' />
+        </Box>
+
+        <Stack direction='row'>
+          <Button colorScheme='blue' size='sm' onClick={() => alert('get started :)')}>
```

Comment:
If the `onClick` handler is going to be implemented in a follow-up PR, add a TODO for it